src/App.js

```diff
@@ -1,12 +1,24 @@
 import React, { Component } from 'react';
-
+import './app.scss';
+import { Button } from 'carbon-components-react';
+import { Content } from 'carbon-components-react/lib/components/UIShell';
+import TutorialHeader from './components/TutorialHeader';
+import { Route, Switch } from 'react-router-dom';
+import LandingPage from './content/LandingPage';
+import RepoPage from './content/RepoPage';
 class App extends Component {
   render() {
     return (
-      <div>
-        Hello Carbon! Well, not quite yet. This is the starting point for the
-        Carbon tutorial.
-      </div>
+      <>
+        <TutorialHeader />
+        <Content>
+          <Button>Button</Button>
```

Comment:
Per instructions, the button component should be removed.

Reply:
Hi,

I already deleted the button code and committed it. Please check it again. Thanks.